concent_api/middleman/tests/test_middleman.py

```diff
@@ -77,45 +77,48 @@ def test_that_middleman_is_created_with_default_params(self):  # pylint: disable
 class TestMiddleManServer:
     patcher = None
     crash_logger_mock = None
+    internal_port = None
+    external_port = None
+    middleman = None
 
     @pytest.fixture(autouse=True)
-    def middleman_setup(self, unused_tcp_port_factory, event_loop):
+    def _middleman_setup(self, unused_tcp_port_factory, event_loop):
         self.patcher = mock.patch("middleman.middleman_server.crash_logger")
         self.crash_logger_mock = self.patcher.start()
         self.internal_port, self.external_port = unused_tcp_port_factory(), unused_tcp_port_factory()
         self.middleman = MiddleMan(internal_port=self.internal_port, external_port=self.external_port, loop=event_loop)
         yield self.internal_port, self.external_port
         self.patcher.stop()
 
-    def test_that_if_keyboard_interrupt_is_raised_application_will_exit_without_errors(self, middleman_setup):
+    def test_that_if_keyboard_interrupt_is_raised_application_will_exit_without_errors(self, _middleman_setup):
```

Comment:
remove `_middleman_setup` from here and the rest of test methods

Reply:
Done